libraries/AP_TECS/AP_TECS.h

```diff
@@ -314,6 +292,14 @@ class AP_TECS : public AP_SpdHgtControl {
 
     float _distance_beyond_land_wp;
 
+    // internal variables to be logged
+    struct {
+        float SKE_weighting;
+        float SPE_error;
+        float SKE_error;
+        float SEB_delta;
+    } logging;
+    
```

Comment:
old struct was a logging structure as per old log method. New struct is just the vars that need to be preserved for logging.